include/sparrow/layout/primitive_array.hpp

```diff
@@ -55,6 +53,21 @@ namespace sparrow
         using iterator_tag = std::random_access_iterator_tag;
     };
 
+    /**
+     * Array of values having the same binary size.
+     *
+     * The \c primitive_array class represents an array of values having the same binary size.
+     * The current implementation supports types whose size is known at compile only.
+     *
+     * As the other arrays in sparrow, \c primitive_array<T> provides an API as if it was holding
+     * \c nullable<T> values instead of \c T values.
+     *
+     * Internally, the array contains a validity bitmap and a contiguous memory buffer
+     * holding the values.
+     *
+     * @tparam T the type of the values in the array.
+     * @see https://arrow.apache.org/docs/dev/format/Columnar.html#fixed-size-primitive-layout
```

Comment:
Move this line in the description?

Reply:
This is usually easier to find reference at the end in a dedicated function rather than drown in the middle of the description. Even if the description is quite short for now, it may grow.

Reply:
I meant at the end of the description but before the section about the parameters (which will always be apart anyway in doxygen generated doc)